activerecord/lib/active_record/migration/command_recorder.rb

```diff
@@ -164,6 +164,16 @@ def invert_remove_index(args)
       alias :invert_add_belongs_to :invert_add_reference
       alias :invert_remove_belongs_to :invert_remove_reference
 
+      def invert_change_column_default(args)
+        table, column, options = *args
+
+        unless options && options.is_a?(Hash) && options[:from] && options[:to]
```

Comment:
Good question. I would expect that it should, but [the test](https://github.com/rails/rails/pull/20018/files#diff-9531a83a96f7d0e6d2611e8494f50c96R172) didn't fail. Let me look further into this.